libmscore/cmd.cpp

```diff
@@ -3162,6 +3162,8 @@ void Score::cmd(const QAction* a)
             { "add-marcato",                [this]{ addArticulation(SymId::articMarcatoAbove);                  }},
             { "add-sforzato",               [this]{ addArticulation(SymId::articAccentAbove);                   }},
             { "add-trill",                  [this]{ addArticulation(SymId::ornamentTrill);                      }},
+            { "add-stringsUpBow",           [this]{ addArticulation(SymId::stringsUpBow);                       }},
```

Comment:
We should use camelCase here since others don't use it. Except this, it's good, I'll merge it and change it